exercise/static/exercise/personalized_points_goal.js

```diff
@@ -0,0 +1,146 @@
+$(document).ready(function() {
+    const $pointsGoalForm = $('#pointsGoalForm');
+    const $inputField = $('#id_personalized_points_goal_input');
+    $inputField.focus();
+    $pointsGoalForm.on('submit', function(event) {
+        event.preventDefault();
+
+        // Validate input
+        const inputValue = $inputField.val().trim();
+
+        const isNumber = !isNaN(inputValue) && inputValue !== '';
+        const isPercentage = inputValue.endsWith('%') && !isNaN(inputValue.slice(0, -1));
+
+        if (!isNumber && !isPercentage) {
+            $('#validation-alert').show();
+            setTimeout(function() {
+                $('#validation-errors-alert').hide();
+            }, 5000);
```

Comment:
This shows a message `#validation-alert` and hides `#validation-errors-alert`.
Meaning, `#validation-alert` never gets hidden.